htdocs/xoops_lib/modules/protector/library/HTMLPurifier/CSSDefinition.php

```diff
@@ -370,6 +370,19 @@ protected function doSetupProprietary($config)
         );
         $this->info['page-break-inside'] = new HTMLPurifier_AttrDef_Enum(array('auto', 'avoid'));
 
+        $border_radius = new HTMLPurifier_AttrDef_CSS_Composite(
+            array(
+                new HTMLPurifier_AttrDef_CSS_Percentage(true), // disallow negative
+                new HTMLPurifier_AttrDef_CSS_Length('0') // disallow negative
+            ));
```

Comment:
__[code_sniffer]__
* Multi-line function call not indented correctly; expected 8 spaces but found 12
* Closing parenthesis of a multi-line function call must be on a line by itself

<!-- Comment by [SideCI](https://sideci.com) -->